modules/metrics/prometheus/src/main/scala/higherkindness/mu/rpc/prometheus/PrometheusMetrics.scala

```diff
@@ -33,9 +33,9 @@ import io.prometheus.client._
  * {prefix}_messages_sent{labels=classifier,service,method} - Counter
  * {prefix}_messages_received{labels=classifier,service,method} - Counter
  * {prefix}_calls_header{labels=classifier} - Histogram
- * {prefix}_calls_total{labels=classifier,method,status} - Histogram
+ * {prefix}_calls_total{labels=classifier,methodType,status} - Histogram
```

Comment:
This was confusing since we had in the messages sent/received metrics the same label name but with different values.